gradle.properties

```
@@ -0,0 +1,15 @@
+org.gradle.caching=true
+group=io.ballerina.lib
+version=1.0.0-SNAPSHOT
```

```suggestion
version=1.0.0-SNAPSHOT
```

Is this version correct?

---

Changed to 0.8.0